doc/source/api/satpy.rst

```diff
@@ -5,6 +5,7 @@ Subpackages
 -----------
 
 .. toctree::
+   :maxdepth: 4
```

Comment:
What is the difference in the result between the default and 4?

Reply:
No clue, all I did was run `sphinx-apidoc` as documented.

Reply:
Ah ok. So this is a generated change. Nevermind then.